.github/workflows/skywalking.yaml

```diff
@@ -676,7 +676,9 @@ jobs:
           name: docker-images-11
           path: docker-images
       - name: Load docker images
-        run: find docker-images -name "*.tar" -exec docker load -i {} \;
+        run: |
+          find docker-images -name "*.tar" -exec docker load -i {} \;
+          find docker-images -name "*.tar" -exec rm {} \;
```

Comment:
@ZhuWang1112 We helped you to fix this. New downloads seem to be over the volume size limitation, which causes the failure.

Reply:
Really appreciate your help, and I will be mindful of the volume size limitation in the future.